.github/workflows/release.yaml

```diff
@@ -118,15 +114,26 @@ jobs:
         if: matrix.config.os == 'windows-latest'
         run: compress-archive ./dist/iohk_wallet chain-wallet-libs-${{ steps.get_version.outputs.VERSION }}-${{ matrix.config.target }}.zip
 
+      - name: Get file extension
+        id: get_file_extension
+        run: |
+          if [ ${{ matrix.config.os }} = "windows-latest" ]
+          then
+            echo "::set-output name=EXT::zip"
+          else
+            echo "::set-output name=EXT::tar.gz"
+          fi
+        shell: bash
+
       - name: Upload binaries to release
-        uses: svenstaro/upload-release-action@v1-release
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: chain-wallet-libs-${{ steps.get_version.outputs.VERSION }}-${{ matrix.config.target }}.*
+          upload_url: ${{ needs.initial_release.outputs.upload_url }}
+          asset_path: chain-wallet-libs-${{ steps.get_version.outputs.VERSION }}-${{ matrix.config.target }}.${{ steps.get_file_extension.outputs.EXT }}
```

Comment:
It would be better if all output names used the same capitalization, like `upload_url` above. All caps looks odd in these expressions.